src/components/Input/Type/Radio.js

```diff
@@ -0,0 +1,102 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { connectTheme } from '../../../utils';
+
+const isLimPrim = function(input) {
+  return { string: '', number: '' }.hasOwnProperty(typeof input);
+};
+
+const optTypeCheck = function(option) {
+  if (Array.isArray(option)) return isLimPrim(option[0]) ? 'array' : false;
+  else return isLimPrim(option) ? 'lim-prim' : false;
```

Comment:
what is `lim-prim`?

Reply:
a subset of primitives -> number and string

Reply:
Why is the output polymorphic?